scripts/apps/authoring/authoring/directives/SendItem.tsx

```diff
@@ -909,7 +911,9 @@ export function SendItem($q,
                     scope.currentUserAction === ctrl.userActions.duplicate_to) {
                     var lastDestination = scope.destination_last[scope.currentUserAction];
 
-                    if (lastDestination) {
+                    if (appConfig.ui && appConfig.ui.defaultStage != null && appConfig.ui.defaultStage === 'working') {
+                        stage = scope.stages.find((stg) => stg.working_stage);
```

Comment:
I think there should be also handling for the setting being incoming so if set it behaves consistently:
```typescript
if (appConfig.ui && appConfig.ui.sendDefaultStage != null) {
    stage = scope.stages.find((stg) => appConfig.ui.defaultStage === 'working' ? stg.working_stage : stg.incoming_stage));
} ...
```